qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst

```diff
@@ -93,6 +107,10 @@ Optional fields for centralized Qiita by portal
 Prep information file
 ---------------------
 
+The *prep information file* will describe the sequencing technology or analytical
```

Comment:
I threw in 'analytical chemistry' to include proteomics and metabolomics. Thoughts?
I could also say:
`The *prep information file* will describe the technology used to generate this data type, including sequencing, proteomics, metabolomics, etc.`